app/components/candidate_interface/links_to_previous_applications_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<% submitted_applications.each do |application_form| %>
+  <%= govuk_link_to ordinalize(application_form), candidate_interface_application_review_previous_path(application_form.candidate.id, application_form.id) %>
```

Comment:
Optional: this reads a bit awkwardly, perhaps:

```suggestion
  <%= govuk_link_to ordinalize(application_form), candidate_interface_review_previous_application_path(application_form.candidate.id, application_form.id) %>
```

Reply:
That makes sense. I was trying to stick with the convention, but this reads so much more clearly 👌